frontend/src/views/LandingPage/GoalBlock.vue

```diff
@@ -1,26 +1,27 @@
 <template>
   <div>
-    <h2 class="text-h4 font-weight-black">Les enjeux de la restauration collective</h2>
-    <p class="text-left mt-8">
+    <h2 class="text-h4 font-weight-black mb-8">L'enjeu de notre mission</h2>
+
+    <p class="text-left">
       La restauration collective représente près de
       <b>4 milliards de repas servis par an</b>
-      dans plus de 90 000 sites de restauration, des cantines scolaires, des hopitaux, des EHPAD, des entreprises, des
-      administrations... Agir sur la composition des assiettes est un levier fort pour améliorer l'impact sur notre
-      santé et sur l'environnement tout en favorisant le développement des filières d'agriculture biologique et de
-      qualité. Les mesures en faveur d'une alimentation plus durable peuvent contribuer à modifier les comportements
-      alimentaires notamment chez les plus jeunes.
+      dans plus de 90 000 de cantines, d’hopitaux, d’EPHAD, d’entreprises...
     </p>
-
-    <v-card elevation="0" color="secondary lighten-3" class="mt-8 pa-md-10">
-      <v-card-text>
-        <video style="width: 100%; background: #333;" poster="/static/images/video-poster-intro.webp" controls>
-          <source
-            type="video/mp4"
-            src="https://cellar-c2.services.clever-cloud.com/ma-cantine-egalim/videos/presentation-egalim.mp4"
-          />
-          Votre navigateur ne peut pas afficher des vidéos.
-        </video>
-      </v-card-text>
-    </v-card>
+    <p class="text-left">
+      Agir sur la composition des assiettes est un levier fort pour améliorer l'impact sur notre santé et sur
+      l'environnement tout en influançant positivement les filières d'agriculture biologique et de qualité.
+    </p>
+    <p class="text-left">
+      Le cadre réglementaire et les deux lois EGAlim (2018) et la loi Climat et Résilience (2021) ont fixé des objectifs
```

Comment:
```suggestion
      Le cadre réglementaire et les deux lois, EGAlim (2018) et Climat et Résilience (2021), ont fixé des objectifs
```
P-e j'ai mal compris le sens original - il implique que EGAlim c'est une combinasion de deux lois (et il y a une troisième, Climat et Résilience) ?

Reply:
Pas tout à fait, en gros il y a deux lois qui déterminent ce qu'il faudra faire pour les gestionnaires de restauration collective. J'ai fait un commit pour reformuler le texte ici : https://github.com/betagouv/ma-cantine/pull/1504/commits/341de8f11447edd16959f2ef162c85f0c87b6054